questions/lookat.md

```diff
@@ -7,7 +7,7 @@ _This method describes how to check whether the player / an entity is **looking
 in Java and later Bedrock versions this can actually be achieved in a single command, thanks to the versatility of the `execute` command. We'll use the following subcommands to achieve our goal:
 
 - `as @a at @s` to modify the execution entity and position.  
-- `anchored eyes` to move the execution position up to the players eyes.  
+- `anchored eyes` to move the execution position up to the players eyes. See this related bug [MC-169665](https://bugs.mojang.com/browse/MC-169665)
 - `facing <entity / coordinates>` to change the execution rotation to be facing our object / entity of desire  
 - `positioned ^ ^ ^1` to move 1 block in the direction of the object  
 - `anchored feet` to move the anchor back down to the players feet (due to a bug in the game which would otherwise apply the eye height modification with each position change, which we don't want, as well as to be able to check for the players existence in the last subcommand).
```

Comment:
talking about bugs, the one this is referring to (https://bugs.mojang.com/browse/MC-124140) seems to have been fixed quite a while ago actually. I can't confirm that right now, but if you could make sure that it would work without that part, that that would be great.

Reply:
So I change the link to the one you provided? Or I keep it as you said in the other comment

Reply:
> So I change the link to the one you provided? Or I keep it as you said in the other comment

Plagiatus is referring to a different bug than the one you linked to.
The bug was that after `anchored eyes positioned ^ ^ ^` you apply `positioned ^ ^ ^` again, `anchored eyes` will be applied again and the command execution position will be shifted higher, instead of remaining at the player's eye level.

@Plagiatus, yes, that bug has been fixed and now you don't need to add `anchored feet`.

Reply:
I just added that is fixed in the current version of the game, and can be omited

Reply:
yeah, it's good as it is now, thank you.